packages/template-retail-react-app/app/hooks/use-add-to-cart-modal.js

```diff
@@ -110,10 +114,7 @@ export const AddToCartModal = () => {
                                     viewType: 'small',
                                     selectedVariationAttributes: variant.variationValues
                                 })?.images?.[0]
-                                const lineItemPrice =
-                                    productItems?.find(
-                                        (item) => item.productId === variant.productId
-                                    )?.basePrice * quantity
+                                const {basePrice, discountPrice} = getDisplayPrice(product)
```

Comment:
```suggestion
                                const {basePrice: lineItemBasePrice, discountPrice: lineItemDiscountPrice} = getDisplayPrice(product)
```
nit: this would make it more clear at a glance whether it's price per item or total price